src/GenSymIO.chpl

```diff
@@ -981,6 +1055,7 @@ module GenSymIO {
                 }
             }
         } catch e: FileNotFoundError {
+              writeln("HELLO");
```

Comment:
Is this a debug message?

Reply:
ROTFL, sure is! Cleanup in aisle 5 is needed on this!

Reply:
fixed